android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -299,11 +312,10 @@ public static void setUsingTestConfiguration(boolean shouldUseTestConfiguration)
         testConfigurationFlag = shouldUseTestConfiguration;
     }
 
-    public void clearTestUpdates() {
-        if (isUsingTestConfiguration()) {
-            codePushPackage.clearTestUpdates();
-            removePendingUpdate();
-        }
+    public void clearUpdates() {
+        codePushPackage.clearUpdates();
```

Comment:
The trouble is knowing whether we are in dev mode from the CodePush class, since it is set in the parent project's BuildConfig. The only way I can think of right now is to have them pass in that value to us in the constructor or the call to getBundleUrl. Do you have a better suggestion?